modules/serverless-configuring-kube-rbac-proxy-resources-for-kafka.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * install/kube-rbac-proxy-kafka.adoc
+
+:_content-type: REFERENCE
+[id="serverless-configuring-kube-rbac-proxy-resources-for-kafka_{context}"]
+= Configuring kube-rbac-proxy resources for Kafka
```

Comment:
```suggestion
= Configuring kube-rbac-proxy resources for Knative for Apache Kafka
```

Reply:
Same suggestion: "Knative Kafka".

Reply:
Resolved in favor of "Knative for Apache Kafka".